kconfigs/qemu.conf

```diff
@@ -1,18 +1,9 @@
 CONFIG_TEE=y
 CONFIG_OPTEE=y
 CONFIG_STAGING=y
```

Comment:
i think you can remove `CONFIG_STAGING=y`

Reply:
Yes, thanks